showcase-docs/dynamic-plugins.md

```diff
@@ -45,18 +45,18 @@ These recommended changes to the `package.json` are summarized below:
   ...
   "scripts": {
     ...
-    "export-dynamic": "backstage-cli package export-dynamic-plugin"
+    "export-dynamic": "janus-cli package export-dynamic-plugin"
     ...
   },
   ...
   "dependencies": {
     ...
-    "@backstage/backend-plugin-manager": "npm:janus-idp/backend-plugin-manager@0.0.5-janus.0",
+    "@backstage/backend-plugin-manager": "npm:@janus-idp/backend-plugin-manager@0.0.5-janus.0",
```

Comment:
```suggestion
    "@backstage/backend-plugin-manager": "npm:@janus-idp/backend-plugin-manager@v1.19.6",
```
I believe that it is better to use the tag of the upstream backstage version here.

`@janus-idp/backend-plugin-manager` versions are tagged with the corresponding commit in the upstream backstage repository, and when built from an upstream release tag, with the corresponding backstage release tag.